### ISSUE TITLE
horizon on RPI

### ISSUE THREAD

**Issue by lavvy** [rethinkdb/horizon#817]
hello guys please could you keep an arm compatible docker image for test. A Dockerfile.armhf would be nice. Many of us test rethinkdb on rpi and its pretty fast.

**Comment by dalanmiller** [comment]
Hey @lavvy, Docker just released 1.12 with ARM compatibility I think two weeks ago. Do you have experience in building images for ARM? Can they also be hosted on Docker Hub? 

I'd love to get this going!

**Comment by lavvy** [comment]
Yeah, Daniel its pretty simple. You have to substitute the base os with an
arm substitute.
In this case, it will be
FROM node:5-slim
Will be
FROM hypriot/node:5-slim

And then build in an arm platform like rpi or scaleway ci.

I pretty much built it and it built successfully but i couldn't test it cos
i haven't learnt rethinkdb and horizon too well

**Comment by dalanmiller** [comment]
Awesome! I'm working on it now actually. 

First step is to build a RethinkDB image, there seems to be some support for the `resin/rpi-raspbian:jessie` image as well. I think the Horizon server image will be much simpler.

**Comment by lavvy** [comment]
Ok cool

**Comment by lavvy** [comment]
@dalanmiller were you able to build and test it
